### PR TITLE
Release 1.92

### PR DESCRIPTION
**Pull request by JStuve**
# Pull Request

Relevant Issue(s): Release changes from the past several months

## Description

Release updates from over the past few months.

## Bug or Feature?

- [ ] Bug fix
- [X] New feature

## Fundamentals

- [ ] I have read the [**CONTRIBUTING**](https://github.com/microsoft/vsts-extension-retrospectives/blob/main/CONTRIBUTING.md) document.
- [ ] My code follows the code style of this project.
- [ ] I have updated any relevant documentation accordingly.
- [ ] I have included an update blurb (50 words or less) at the top of `CHANGELOG.md`,
    to be included with the next release.
  - [ ] I have included a link to this PR in that blurb.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Accessibility

- [ ] I have tested my changes on both light and dark themes.
- [ ] I have tested my changes on both mobile and desktop views, when needed.
- [ ] I have included image descriptions and aria-labels where I can.

### CODE REVIEW

**Comment by JStuve** [comment]
@polatengin Ready to release the changes. We will most likely need an new access token for the CD pipeline.

File: RetrospectiveExtension.Frontend/vss-extension-prod.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "manifestVersion": 1,
   "id": "team-retrospectives",
   "publisher": "ms-devlabs",
-  "version": "1.91.1",
+  "version": "1.92.0",
```
Comment (polatengin) [inline RetrospectiveExtension.Frontend/vss-extension-prod.json:5]: shouldn't it be `1.92.1`?

**Review by polatengin — CHANGES_REQUESTED** [review]
I left a minor comment

File: RetrospectiveExtension.Frontend/vss-extension-prod.json (hunk 1 above)
Comment (JStuve) [reply RetrospectiveExtension.Frontend/vss-extension-prod.json]: Yup, thanks, increased the number

**Review by polatengin — APPROVED** [review]
👍

**Pull request merged**